src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package service.impl;
+
+import db.StorageImpl;
+import service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    @Override
+    public String createReport() {
+        StringBuilder report = new StringBuilder("fruit,quantity");
+        StorageImpl.fruitStorage.forEach((key, value) -> report
+                .append(System.lineSeparator())
+                .append(key)
+                .append(",")
```

Comment:
"fruit,quantity" also can be constant
```suggestion
                .append(SEPARATOR)
```